client/components/grouped-select-control/index.tsx

```diff
@@ -5,81 +5,62 @@ import React, { useRef, useState } from 'react';
 import { Icon, check, chevronDown, chevronUp } from '@wordpress/icons';
 import classNames from 'classnames';
 import { __ } from '@wordpress/i18n';
-import { useSelect } from 'downshift';
+import { useSelect, UseSelectState } from 'downshift';
 
 /**
  * Internal Dependencies
  */
 import './style.scss';
+import { OnboardingFields } from 'wcpay/onboarding-prototype/types';
 
-export interface Item {
+export interface ListItem {
+	type: string;
```

Comment:
`type` is not used anywhere, as `group` and `items` is used to differentiate them.

Reply:
Yes. I was aiming to be more declarative about the intent, but I refactored things in https://github.com/Automattic/woocommerce-payments/pull/6167/commits/204fa68c457dc0fcaabdb733439b20747fef706c